setup.py

```diff
@@ -26,15 +26,9 @@
 import subprocess
 import textwrap
 
-
-if sys.version_info[:2] < (3, 5):
```

Comment:
In principle this is not necessary, assuming the user is running pip>=9.0.0.

Reply:
not everyone is using `pip` to install ....

we should keep this, clear error message helps

Reply:
Changed